app/helpers/entities_helper.rb

```diff
@@ -219,4 +219,25 @@ def entity_tabs(entity, active_tab)
       end.reduce(:+)
     end
   end
+
+  def entity_interlocks_header_for(e)
+    title, subtitle = entity_interlocks_title_and_subtitle_for(e)
+    content_tag(:div, id: "entity-interlocks-header") do
+      content_tag(:div, title, id: "entity-interlocks-title") +
+        content_tag(:div, subtitle, id: "entity-interlocks-subtitle")
+    end
+  end
+
+  private
+
+  def entity_interlocks_title_and_subtitle_for(e)
```

Comment:
cool. make me think how useful having _presenters_ would be

Reply:
+1